src/Widgets/SaveDialog.vala

```diff
@@ -85,6 +85,10 @@ namespace Screenshot.Widgets {
             /// TRANSLATORS: %s represents a timestamp here
             var file_name = _("Screenshot from %s").printf (date_time);
 
+            if (this.scale_factor > 1) {
+                file_name += " @2";
```

Comment:
While it's not common to have factors other than 2, we should actually use the scaling factor here instead of hardcoding `2`.